### ISSUE TITLE
cannotBeEmpty() is not applicable to BooleanNodeDefinition.

### ISSUE THREAD

**Issue by ahmed-bhs** [jdewit/AvroCsvBundle#26]
[Symfony\Component\Config\Definition\Exception\InvalidDefinitionException]
  ->cannotBeEmpty() is not applicable to BooleanNodeDefinition.

**Comment by Saint-Cyr** [comment]
Having the same problem with SF3.4

**Comment by Pasuvan** [comment]
@ahmed-bhs  and @Saint-Cyr Having the same problem, Any one resolved above issue Can you give me a solution.

**Comment by MisatoTremor** [comment]
What version are you using? I can't reproduce this error and none of the release versions > v0.1.0 have a BooleanNode.

**Comment by Pasuvan** [comment]
@MisatoTremor I was upgrade symfony3.4 to Flex. I was update composer above issue happend.
Can you please tell me which bundles version do you want.

**Comment by MisatoTremor** [comment]
I'd like to know the version of AvroCsvBundle you are using so I can try to reproduce the issue.
I updated the Travis CI matrix just last night and even 0.2 did not return such an error on either Symfony 2.8, 3.4 or 4

**Comment by Pasuvan** [comment]
I upgraded symfony3.4 to Flex on the time Symfony/config 4.2.9 also upgraded. I think that this issue happened .

**Comment by MisatoTremor** [comment]
I tried 4.2.9 of `symfony/config` with the lastest version (0.4.0) of this bundle and it worked without any error.

**Comment by oliverfehmel** [comment]
Same problem here.

Versions:

symfony/config 4.2.9
avro/csv-bundle: 0.4.0

It seems the problem is caused by Avro\CaseBundle\DependencyInjection\Configuration Line 28. This line throws an InvalidDefinitionException.

**Comment by MisatoTremor** [comment]
Thanks for your report, the problem is not this bundle, but AvroCaseBundle.

I have fixed that error in my fork: [link](https://github.com/MisatoTremor/AvroCaseBundle/commit/0ccaad2bf4232bdab3904850ff96a01fea1df8dd#diff-850942b3ba24ab03a40aaa81b6152852L28) 3 years ago, thus forgetting about it.

Unfortunately the PR containing that fix (jdewit/AvroCaseBundle#3) has not been merged and I don't think @jdewit does maintain these anymore. I already thought about renaming my fork and using that package as a dependency for CSV.

You could use my repository in your composer.json in the meantime like this
```json
    "require": {
        "avro/case-bundle": "^0.4.0",
        "avro/csv-bundle": "^0.4.0",
    },
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/MisatoTremor/AvroCaseBundle"
        }
    ],
```
